web/packages/shared/components/UnifiedResources/UnifiedResources.tsx

```diff
@@ -286,12 +284,12 @@ export function UnifiedResources(props: UnifiedResourcesProps) {
   );
 
   return (
-    <FeatureBox
+    <div
       className="ContainerContext"
```

Comment:
Actually, we could remove `ContainerContext` from here and let the parent set it, since it is mainly used to limit the width of the search input (and we won't have that thing in Connect). But I think it is more convenient to keep it as is.